tr1d1um.yaml

```diff
@@ -28,57 +28,51 @@ region: "east"
 # (Optional)
 flavor: "mint"
 
+prometheus:
+  defaultNamespace: xmidt
+  defaultSubsystem: tr1d1um
+  constLabels:
+    development: "true"
+  handler:
+    maxRequestsInFlight: 5
+    timeout: 5s
+    instrumentMetricHandler: true
 
-##############################################################################
-# WebPA Service configuration
-##############################################################################
-
-# For a complete view of the service config structure, 
-# checkout https://godoc.org/github.com/xmidt-org/webpa-common/server#WebPA
-
-########################################
-#   Primary Endpoint Configuration
-########################################
-
-# primary provides the configuration for the main server for this application
-primary:
-  address: ":6100"
-
-########################################
-#   Health Endpoint Configuration
-########################################
-
-# health defines the details needed for the health check endpoint.  The
-# health check endpoint is generally used by services (like AWS Route53
-# or consul) to determine if this particular machine is healthy or not.
 health:
-  address: ":6101"
-
-########################################
-#   Debugging/Pprof Configuration
-########################################
-
-# pprof defines the details needed for the pprof debug endpoint.
-# (Optional)
-pprof:
```

Comment:
since we deleted this here, we should add a new key under the servers config to provide this configuration I think